### PR TITLE
refactor!(breaking-change): bump dependencies to appium2

### PR DESCRIPTION
**Pull request by KazuCocoa**
should we wait for this change until win app driver supports w3c spec one...?

----

`breaking-change` bump major version by the semantic release action

### CODE REVIEW

**Review by mykola-mokhnach — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -72,12 +72,16 @@
...
+    "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
-    "eslint-config-appium": "^4.0.1",
+    "eslint": "^7.32.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-mocha": "^9.0.0",
+    "eslint-plugin-promise": "^6.0.0",
     "gulp": "^4.0.0",
     "pre-commit": "^1.2.2",
     "semantic-release": "^19.0.2",
```
Comment (mykola-mokhnach) [inline package.json:87]: wd should replaced with webdriverio as a part of this change and the corresponding e2e tests must be updated

Comment (mykola-mokhnach) [reply package.json:87]: otheriwise we should disable failing e2e tests and anyway exclude wd from dependencies list

**Comment by mykola-mokhnach** [comment]
I assume the CI wants to be upgraded to use NPM 14.x

**Comment by mykola-mokhnach** [comment]
> should we wait for this change until win app driver supports w3c spec one...?

NO, we should not. JWProxy sill supports both protocols. Btw remote debugger in xcuitest driver also only in JWP mode

**Comment by KazuCocoa** [comment]
Updated an e2e test with webdriverio (a very simple one). The other will be another pr...

File: .azure-pipelines.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # https://github.com/actions/virtual-environments
 variables:
-  DEFAULT_NODE_VERSION: "10.x"
+  # v14 fails on Win env: https://github.com/actions/setup-node/issues/411
```
Comment (mykola-mokhnach) [inline .azure-pipelines.yml:3]: have you tried a workaround at https://github.com/actions/setup-node/issues/411#issuecomment-1025543081

I assume we should still use v14 as this is the minimum supported one

File: test/e2e/driver-e2e-specs.js [hunk 3]
```diff
@@ -10,6 +10,17 @@ chai.use(chaiAsPromised);
 const TEST_PORT = 4788;
 const TEST_HOST = 'localhost';
 
+const TEST_CAPS = {
+  platformName: 'Windows',
+  app: 'Microsoft.WindowsCalculator_8wekyb3d8bbwe!App'
```
Comment (mykola-mokhnach) [inline test/e2e/driver-e2e-specs.js:15]: should it be 'appium:app' ?

**Comment by KazuCocoa** [comment]
mm, breaking change release did not occur by https://github.com/appium/appium-windows-driver/commit/dad29b1d6c57150bfa04bbc7b57a9806940ae837 as well....
Let me check further...
https://github.com/semantic-release/semantic-release/blob/8fda7fd423d24e7b425fbee83790f49dd0478e2d/docs/support/FAQ.md#how-can-i-change-the-type-of-commits-that-trigger-a-release

**Pull request merged**